### PR TITLE
Bring back the topic status indication message

### PR DESCRIPTION
**Pull request by allenwq**
screen shot 2017-10-02 at 11 06 21 am

### CODE REVIEW

File: config/locales/en/course/forum/topics.yml [hunk 1]
```diff
@@ -52,3 +52,8 @@ en:
           tag: 'Unhide'
           success: "The topic was successfully unhidden."
           failure: "The topic could not be unhidden."
+        resolved:
+          message: 'This question topic has been resolved.'
+        unresolved:
+          message: 'This question topic is unresolved.'
+          tag: 'Mark any posts as the answer to resolve the question.'
```
Comment (fonglh) [inline config/locales/en/course/forum/topics.yml]: If they can't mark multiple posts as the answer, it should be `Mark any post ...` instead.

Comment (allenwq) [reply config/locales/en/course/forum/topics.yml]: Actually multiple posts can be marked, but they can only mark one at a time. In that case which one do you think is better?

Comment (fonglh) [reply config/locales/en/course/forum/topics.yml]: How about `Mark helpful posts as answers to resolve this question`?

Comment (allenwq) [reply config/locales/en/course/forum/topics.yml]: OK 👍  will do the change.

**Pull request merged**